geowebcache/wms/src/main/java/org/geowebcache/io/ImageEncoderImpl.java

```diff
@@ -94,7 +93,9 @@ public ImageWriteParam prepareParameters(
                         // we can control quality here
                         params.setCompressionQuality(compressionRate);
                     }
-                } else if (writer instanceof PNGImageWriter) {
+                    // Use class name without import for JDK 11 compatibility
+                } else if ("com.sun.imageio.plugins.png.PNGImageWriter"
```

Comment:
Technically one should go through all the superclasses and compare names (if it's a subclass of PNGImageWriter maybe?), but I guess this one is more than adequate.

Reply:
Technically true, although given its a sun internal class in the first place, with no official subclasses, I would hope no one is subclassing it.

Reply:
Indeed